### PR TITLE
[RDAPI-25902] Add documentation for Nov YAML PS support

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Docs changes for YAML PS support tickets

## Deploy preview link

https://deploy-preview-2171--axway-open-docs.netlify.app/docs/apim_relnotes/20211130_apimgr_relnotes/#yaml-support-in-policy-studio
https://deploy-preview-2171--axway-open-docs.netlify.app//docs/apigtw_devops/team_dev_practices/#when-to-use-team-development

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by lcattolico — APPROVED** [review]

File: content/en/docs/apim_relnotes/20211130_apimgr_relnotes.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Policy Studio has initial support for YAML-based policies with known limitations
 * Creating a new project from an API Gateway instance using environmentalized values.
 * Opening a YAML project with manually environmentalized values.
 * Importing fragments with environmentalized values.
+* Copy/Paste options on the Policy tree view.
+* *'Configure API Manager...'* and *'Configure OAuth...'* file menu options.
```
[the remark this answers is not in the document]
Comment (JohnD-Axway) [reply content/en/docs/apim_relnotes/20211130_apimgr_relnotes.md:47]: @lcattolico here it is.
@Bcolemanaxway ^^

[the remark this answers is not in the document]
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20211130_apimgr_relnotes.md:47]: @lcattolico @Bcolemanaxway if you're going to change something be aware that I've pushed a commit, so remember to pull my changes first.

File: content/en/docs/apigtw_devops/team_dev_practices.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ When using team development you must:
 
 The team development workflow requires you to split projects into separate pieces of configuration so that teams can work on them independently. If you do not have a requirement for multiple policy developers to work in parallel, you should not use a team development workflow. Instead you should use the previous workflows in Policy Studio where policy developers typically connect to a running API Gateway instance, download the configuration, make desired changes, and then deploy the configuration back to the API Gateway. This approach encourages use of API Gateway as a design-time repository for policies, in addition to as the runtime for executing policies. This workflow still requires you to create a project in Policy Studio, but there is no requirement to split the project into separate pieces of configuration, as in the team development workflow.
 
+{{< alert title="Note" color="primary" >}}Team Development functionality is not available when using [YAML-based projects](/docs/apim_yamles/).{{< /alert >}}
+
```
Comment (Bcolemanaxway) [inline content/en/docs/apigtw_devops/team_dev_practices.md:23]: @JohnD-Axway  This is where we added the note about Team Dev. Seemed like it needed to be added to the actual docs and not just a release note seeing as we don't ever plan on supporting team dev in YAML.

Comment (andreamussap) [reply content/en/docs/apigtw_devops/team_dev_practices.md:23]: @Bcolemanaxway it doesn't hurt to add to the release as well. Why don't you add to [this list ](https://deploy-preview-2171--axway-open-docs.netlify.app/docs/apim_relnotes/20211130_apimgr_relnotes/#yaml-support-in-policy-studio) ??

* [Team development](/docs/apigtw_devops/team_dev_practices.md)

Comment (Bcolemanaxway) [reply content/en/docs/apigtw_devops/team_dev_practices.md:23]: @andreamussap Added this to the release note!

Comment (JohnD-Axway) [reply content/en/docs/apigtw_devops/team_dev_practices.md:23]: @lcattolico @Bcolemanaxway We need to add the fact we can't import custom filters!

Comment (Bcolemanaxway) [reply content/en/docs/apigtw_devops/team_dev_practices.md:23]: @JohnD-Axway Added it to release notes section!

Comment (JohnD-Axway) [reply content/en/docs/apigtw_devops/team_dev_practices.md:23]: Thanks @Bcolemanaxway

**Pull request merged**